api/src/Router.ts

```diff
@@ -1,10 +1,15 @@
 import KoaRouter from "koa-router";
 
+import * as AuthController from "./controllers/AuthController"
+
 function createRouter() {
   const router = new KoaRouter();
 
+  // Auth Routes
+  router.post("/api/login", AuthController.login)
+
   // Health endpoint
-  router.get("/health", (ctx) => {
+  router.get("/api/health", (ctx) => {
```

Comment:
Why the`/api` prefix? Maybe a good place for versioning, like v1, but this is already the API :D